composables/transaction/transactionSend.ts

```diff
@@ -49,11 +52,14 @@ function execSendAssetHub(item: ActionSend, api, executeTransaction) {
 function execSendEvm(item: ActionSend, executeTransaction: ExecuteTransaction) {
   const { accountId } = useAuth()
 
+  // TODO: add evm safeBatchTransferFrom support
+  const nft = item.nfts[0]
+
   executeTransaction({
-    address: item.collectionId,
+    address: nft.collectionId,
     abi: item.abi as Abi,
     functionName: 'safeTransferFrom',
-    arg: [accountId.value, item.address, item.nftSn],
+    arg: [accountId.value, item.address, nft.sn],
   })
```

Comment:
should the `batchTransfer` be implemented before merging?

Reply:
`batchTransfer`  is not available on `ERC721`

`safeBatchTransferFrom` is added in `ERC1155` 
https://docs.openzeppelin.com/contracts/3.x/erc1155#batch-operations

@vikiival am I missing something ?

Reply:
Sorry, I didn't make it clear. I mean supporting batch transfer on EVM, no matter which method on ERC.

However, I saw multiple NFT transfers on EVM prevented on the UI side. So, it's not that urgent to implement batch transfer for now.


<img width="1039" alt="image" src="https://github.com/user-attachments/assets/20471e49-efe8-4bd3-820b-5cee73a50cd7">